docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
We pre-sign OCSP responses for high traffic domains. For low traffic domains that are generally never accessed, we sign with a delegated OCSP responder. The 30 day requirement  validity period works for us.